CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # 1.46.0
 
+## ✨ Features
+
+* Send hideSplashScreen message to Flagship app when Home App is ready
```

Comment:
Maybe this message is too technical for a `feature` changelog?

Reply:
Maybe something like `When displaying cozy-home from Cozy's native application, the splash screen is now correctly shown during the page loading`